_includes/manuals/1.19/3.6_upgrade.md

```diff
@@ -74,7 +74,8 @@ number from the previous release or "stable" to `{{ page.version }}`:
     deb http://deb.theforeman.org/ stretch {{ page.version }}
     deb http://deb.theforeman.org/ plugins {{ page.version }}
 
-Next upgrade all Foreman packages:
+Next upgrade all Foreman packages.
+During the upgrade it's suggested to update the /etc/foreman/database.yml to the package version. If you used foreman-installer to setup your existing Foreman instance, make sure not to upgrade this file with the version from the package.
```

Comment:
How does one `update the /etc/foreman/database.yml to the package version.`?

Reply:
`apt upgrade` will ask you what to do with config files. This is very similar to running `rpmconf` on RH-based systems but apt will ask you during the upgrade.

Reply:
I'd say in general you probably don't want to replace this, but there can be relevant changes; like recommending the increasing the default pool size. Not sure how to phrase this though.